avm/res/compute/virtual-machine-scale-set/tests/e2e/windows.waf-aligned/main.test.bicep

```diff
@@ -104,7 +104,7 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
         workspaceResourceId: diagnosticDependencies.outputs.logAnalyticsWorkspaceResourceId
       }
     ]
-    encryptionAtHost: true
+    encryptionAtHost: false
```

Comment:
Let's add a comment here as to why this is false

Reply:
Done